R/dataProcessPlots.R

```diff
@@ -260,12 +344,15 @@ dataProcessPlots = function(
                                       legend.size, dot.size.profile, 
                                       ss, s, cumGroupAxis, yaxis.name,
                                       lineNameAxis, groupNametemp, dot_colors)
-      print(profile_plot)
+      
       setTxtProgressBar(pb, i)
+      print(profile_plot)
+      output_plots[["original_plot"]][[paste("plot",i)]] <- profile_plot
     }
+    
     close(pb)
     
-    if (address != FALSE) {
+    if (address != FALSE & isPlotly == FALSE) {
```

Comment:
IIRC address can be chr or lgl, that's why we don't just use if(address). But isPlotly is logical I guess so it's better to use & !isPlotly

Reply:
Commit [better boolean](https://github.com/Vitek-Lab/MSstats/pull/107/commits/6cdc03f21050062af775b02b08b63d47eb8c2480)